pages/en/lb3/Model-definition-JSON-file.md

```diff
@@ -404,6 +406,14 @@ For example:
         See <code>Id</code> property below.
       </td>
     </tr>
+    <tr>
+      <td>generated</td>
+      <td>No</td>
+      <td>Boolean</td>
+      <td>
+        <strong>For auto-migrate usage. And it is only applicable for the id property</strong>. When it sets to <code>true</code>, the value of the property will be generated automatically with the default property type of the database. (e.g integer for MySQL and string for MongoDB)
```

Comment:
suggestion: <strong>For auto-migrate usage. Only applicable for the <code>id</code> property</strong>. When it is set to <code>true</code>, ...etc